### ISSUE TITLE
Support Arkose Labs

### ISSUE THREAD

**Issue by dessant** [dessant/buster#320]
Subscribe to this issue for updates on Arkose Labs support.

**Comment by mohammad708090** [comment]
Unfortunately, the speech system is also inactive and does not work at all

**Comment by dessant** [comment]
The Arkose Labs service is now so hostile that it may be breaking accessibility laws is some jurisdictions. It would be important to raise your concerns at the appropriate government agency in your country.

You can also raise awareness at W3C, they have a working group that documents the inaccessibility of captcha services, and they are receptive to feedback. Let them know about the companies that hinder your access to services and information on the web.

https://www.w3.org/TR/turingtest/

https://github.com/w3c/captcha-accessibility/issues

https://github.com/w3c/apa/issues

**Comment by Cloudef** [comment]
Apparently they are WCAG certified, which makes me question the certification process
https://www.arkoselabs.com/blog/arkose-labs-certified-wcag-compliant/
